reference/configuration/kernel.rst

```diff
@@ -77,6 +81,34 @@ To change this setting, override the
         }
     }
 
+Project Directory
+~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.3
+    The ``getProjectDir()`` method was introduced in Symfony 3.3.
+
+**type**: ``string`` **default**: the directory of the project ``composer.json``
```

Comment:
The genitive is not common for non-living things:

> We normally use the ’s with people, animals though it can also be used with places, organizations and companies (which suggest a group of people).
>
> It is not common to use the ’s with non-living things.

Source: http://www.grammar.cl/rules/genitive-case.htm